### DISCUSSION TITLE
Db prisma

### DISCUSSION THREAD

**Discussion by aethelyn** [ArugaZ/whatsapp-bot#478] [Q&A] [answered]
Bang, cara set DB nya gimana ya? Udh ku kasih link dari mongoDB yang seerti ini "mongodb+srv://username:pass@cluster0.oudejhl.mongodb.net/" masi merah, yg kek gini pun sama merah "mongodb+srv://usern:paaa@cluster0.z8rzocj.mongodb.net/?retryWrites=true&w=majority" pas npx prisma ny, yg bener itu gimana ya?

**Comment by arugaz** [comment, marked as answer]
lupa dikasih db name

```
mongodb+srv://user:pass@mongo.url.mongodb.net/dbname?retryWrites=true&w=majority
```

**Reply by bagushermawan** [reply]
ini fungsinya apa ya bang? waktu ku npx prisma db push error , cuman endingnya bisa jalan sih
![image](https://github.com/arugaz/whatsapp-bot/assets/54248013/c834c9b2-820f-41a4-b01a-120885bcc92c)
![image](https://github.com/arugaz/whatsapp-bot/assets/54248013/4703cae5-5a2a-4f09-ad53-17199ab82824)

**Reply by arugaz** [reply]
sesuaikan url yang ada di schema.prisma dengan variable yang ada di file .env

```
url = env("MONGODB_URL")
```

pada url di .env setelah mongodb.net coba kamu tambahkan
```
/dbname?retryWrites=true&w=majority
```

**Reply by bagushermawan** [reply]
kalo udah gini bang?
![image](https://github.com/arugaz/whatsapp-bot/assets/54248013/732b33c1-5ab1-4946-b406-bf557b2bb56f)

ini fungsinya buat apa ya bang? soalnya walau error, bot wa masih bisa jalan hehe
